### ISSUE TITLE
Cannot select custom time range after using shift selector with auto shift select disabled

### ISSUE THREAD

**Issue by G3CK0za** [isaozler/grafana-shift-selector#31]
Hi @isaozler,

I have been using the latest release for a while now and it's working quite well.

One issue I have noticed is that with auto shift select disabled, if I select a shift using the shift selector I cannot apply a custom time range - it just defaults to the current shift as if auto shift select is enabled.

Could you please take a look when you get a chance?

**Comment by jamesdebono** [comment]
Yes this is a very handy plugin, but +1 for the ability to still select custom time ranges in the normal Grafana dropdown.
